EasyRestProjectNetTeam2/EasyRestPages/SignInPage.cs

```diff
@@ -67,5 +68,11 @@ public void ClickCreateAccountButton()
         {
             _createAccountButton.Click();
         }
+        public void SignInWithValidData(string email, string password)
+        {
+            SendKeysToInputEmail(email);
+            SendKeysToInputPassword(password);
+            HeaderMenuComponent.ClickSignInButton();
```

Comment:
Are you sure that this is the correct button to click? Is it located on the header menu?

Reply:
This SignIn Button has the same Xpath as in Header. So I've little bit cheated. But U are right so I added this element for this page.